src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,74 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String SUPPLY = "supply";
```

Comment:
```suggestion
    private static final String SUPPLY = "supply";
```
константи повинні мати модифікатор доступу private